torchbenchmark/models/sam_fast/__init__.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed according to the terms of the GNU General Public License version 3.
+
+from ...util.model import BenchmarkModel
+from segment_anything_fast.build_sam import sam_model_fast_registry
+from segment_anything_fast.predictor import SamPredictor
```

Comment:
This is really the only difference between this and torchbenchmark/models/sam/__init__.py